mne/viz/tests/test_raw.py

```diff
@@ -25,6 +25,8 @@
 
 def _get_raw():
     raw = io.Raw(raw_fname, preload=True)
+    with warnings.catch_warnings(record=True):
```

Comment:
can you in `master` add a comment on this line or above it saying what the warning was for? it's not immediately obvious to me from the code